README.md

```diff
@@ -56,6 +56,8 @@ The quickest way to install package is through `pip`.
 
 > \$ pip install EMD-signal
 
+In this way you install the old version of PyEMD hosted on PyPi
+
```

Comment:
The wording of this sentence isn't correct. For one, it is the latest officially supported version rather than "the old". If feel strongly that there needs to be an explanation that this package is hosted on pypi then please reprahse it. Otherwise, I think this sentence doesn't add much to the doc and can be deleted.

Reply:
I'll fix the sentence and adjust the paragraph to improve the consistency of the instruction for the installation from source

Reply:
> I'll fix the sentence and adjust the paragraph to improve the consistency of the instruction for the installation from source

Done

Reply:
Sorry, was traveling. Looks good!